static/styles.css

```diff
@@ -166,6 +166,41 @@ h5 {
 .emoji-button {
 	background: none;
 	border: none;
-	font-size: 1.5em;
+	font-size: 1.4em;
 	cursor: pointer;
+
+	width: 1.7em;
+	height: 1.7em;
+	border-radius: 50%;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+}
+
+.highlight_green{
+	background-color:lightgreen;
```

Comment:
shouldn't this class just need the background color? I think the rest of these properties would already be specified by class emoji-button, and including them makes this css class less general.

Unless I'm missing something this overrides "on purpose"

Reply:
I would like to see these styles cleaned up, or I would like to hear that I'm wrong here. 

I think this is all that's keeping the code from getting merged.